packages/atomic/src/components/search/result-template-components/atomic-quickview-sidebar/atomic-quickview-sidebar.tsx

```diff
@@ -86,47 +92,86 @@ const HighlightKeywordsCheckbox: FunctionalComponent<
   </Fragment>
 );
 
-const KeywordsNavigation: FunctionalComponent<
-  Pick<QuickviewSidebarProps, 'i18n'> & {
+const Keywords: FunctionalComponent<
+  Pick<
+    QuickviewSidebarProps,
+    'i18n' | 'onHighlightKeywords' | 'highlightKeywords'
+  > & {
     words: Record<string, QuickviewWordHighlight>;
   }
-> = ({words, i18n}) => {
+> = ({words, i18n, highlightKeywords, onHighlightKeywords}) => {
   return (
     <Fragment>
       {Object.values(words).map((keyword) => {
+        const wordIsEnabled =
+          highlightKeywords.highlightAll &&
+          (highlightKeywords.keywords[keyword.text] === undefined ||
+            highlightKeywords.keywords[keyword.text].enabled === true);
+
         return (
           <div
             key={keyword.text}
-            class="flex items-center w-100 my-4 bg-background border border-neutral rounded-lg overflow-x-auto"
+            class="flex items-center justify-between gap-x-2 my-4 w-100"
           >
-            <div class="flex items-center grow p-4 border-r">
-              <div
-                class="w-5 h-5 flex-none mr-2"
-                style={{backgroundColor: keyword.color}}
-              ></div>
-              <div class="grow mr-2">{keyword.text}</div>
-              <div class="flex-none">{keyword.occurrences}</div>
-            </div>
-            <div class="flex px-2">
-              <atomic-icon-button
-                buttonStyle="text-transparent"
-                class="mr-2 border-0"
-                labelI18nKey="next"
-                tooltip={i18n.t('next')}
-                icon={ArrowDown}
-                clickCallback={() => {
-                  keyword.navigateForward();
-                }}
-              ></atomic-icon-button>
-              <atomic-icon-button
-                buttonStyle="text-transparent"
-                class="border-0"
-                labelI18nKey="previous"
-                tooltip={i18n.t('previous')}
-                icon={ArrowUp}
-                clickCallback={() => keyword.navigateBackward()}
-              ></atomic-icon-button>
+            <div
+              class={`flex grow items-center bg-background border border-neutral rounded-lg overflow-x-auto ${
+                !wordIsEnabled ? 'pointer-events-none opacity-50' : ''
```

Comment:
What is `pointer-events-none` for in this case?

Reply:
So that there is no mouse interaction when a word is disabled. For example, this disables all visual hover effect on buttons.

Reply:
does this prevent the user from selecting the text and copying it?